waspc/packages/deploy/src/providers/fly/index.ts

```diff
@@ -20,6 +20,10 @@ class FlyCommand extends Command {
 			.option('--initial-cluster-size <initialClusterSize>', 'flyctl postgres create option', '1')
 			.option('--volume-size <volumeSize>', 'flyctl postgres create option', '1');
 	}
+	addLocalBuildOption(): this {
+		return this.option('--build-server-locally', 'force the server Docker container to build locally', false)
+			.option('--build-client-locally', 'force the client Docker container to build locally', false);
```

Comment:
Just curious, how is it that we have two commands, and not just one `--build-locally` that would build everything locally? When might one want to build one thing locally and other thing not?

Reply:
Unsure, I can collapse it to one option for now and we can split if needed.